openfecli/tests/commands/test_gather/test_septop_full_results_dg_.csv

```diff
@@ -1,5 +1,5 @@
 ,ligand,DG (kcal/mol),uncertainty (kcal/mol)
-0,1,-0.67500000000000027,0.66426651277932103
-1,25,1.3250000000000119,1.3119641763401924
-2,7a,-0.87500000000000566,0.9034655499796348
-3,7b,0.22499999999999401,0.90346554997963024
+0,1,-0.71703997416806253,0.66060368310736228
```

Comment:
@hannahbaumann this PR is to evaluate the actual value differences from fixing the rounding bug - a follow-up PR that checks the rounding precision on output is here: https://github.com/OpenFreeEnergy/openfe/pull/1697